common/safety-push-metrics/src/counters.rs

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) The Libra Core Contributors
```

Comment:
This file would benefit substantially with more documentation describing the types and how they are to be used
As well as the formatting

Reply:
Was planning on doing so before landing. Just didn't want do document a bunch of interfaces which would've been removed in code review.

Reply:
Done